packages/app/src/components/Settings/Settings.js

```diff
@@ -194,59 +214,114 @@ const Settings = (props) => {
+              <div
+                style={
+                  showEditModal
+                    ? {
+                        display: 'none',
+                      }
+                    : {
+                        position: 'relative',
+                      }
+                }
+              >
+                <DatePicker
+                  name="openedOn"
+                  value={newGrantCycle.openedOn}
+                  selected={newGrantCycle.openedOn}
+                  onChange={handleStartDateCreation}
+                  placeholderText="mm/dd/yyyy"
+                />
+              </div>
             </span>
           </div>
           <div className="settings__input-block">
             <p className="settings__input-label">End Date:</p>
             <span className="settings__input">
-              <input
-                value={newGrantCycle.closedOn}
-                name="closedOn"
-                onChange={handleChange}
-                type="date"
-              />
+              <div
+                style={
+                  showEditModal
+                    ? {
+                        display: 'none',
+                      }
+                    : {
+                        position: 'relative',
+                      }
+                }
+              >
+                <DatePicker
+                  name="closedOn"
+                  value={newGrantCycle.closedOn}
+                  selected={newGrantCycle.closedOn}
+                  onChange={handleEndDateCreation}
+                  placeholderText="mm/dd/yyyy"
+                />
+              </div>
             </span>
           </div>
           <div className="settings__input-block">
             <p className="settings__input-label">Cycle Name:</p>
             <span className="settings__input">
-              <input
-                value={newGrantCycle.name}
-                name="name"
-                onChange={handleChange}
-                type="text"
-              />
+              <div
+                style={
+                  showEditModal
+                    ? {
+                        display: 'none',
+                      }
+                    : {
+                        position: 'relative',
+                      }
+                }
+              >
+                <input
+                  value={newGrantCycle.name}
+                  name="name"
+                  onChange={handleChange}
+                  type="text"
+                />
+              </div>
             </span>
           </div>
         </div>
-        <button
-          ref={createButton}
-          disabled={disableButton}
-          onClick={handleCreate}
-          className="settings__button"
+        <div
           style={
-            disableButton
-              ? { color: 'gray', 'border-color': 'gray', 'font-weight': 'bold', 'opacity': '1' }
+            showEditModal
+              ? {
+                  display: 'none',
+                }
```

Comment:
Same goes for here even though it existed before.

Having `.hidden { display: 'none' }` somewhere in the CSS should allow us to do this (or similar) in all these places:

```javascript
const editModalHiddenClass = showEditModal ? "" : ".hidden";

// Later...

<div className={ editModalHiddenClass }>

```